eng/common/pipelines/templates/steps/save-package-properties.yml

```diff
@@ -61,10 +61,11 @@ steps:
   - ${{ else }}:
       - task: Powershell@2
         displayName: Save package properties
+        condition: succeeded()
         inputs:
           filePath: ${{ parameters.ScriptDirectory }}/Save-Package-Properties.ps1
           arguments: >
             -ServiceDirectory '${{parameters.ServiceDirectory}}'
             -OutDirectory '${{ parameters.PackageInfoDirectory }}'
-            -AddDevVersion:$${{ eq(variables['SetDevVersion'],'true') }}
```

Comment:
AddDevVersion is a switch, and we can't pass a boolean to override the value of switch. DevOps also does not process expression evaluation at this level, so I have changed it to use variable value to pass the bool flag to script.